tmt/cli.py

```diff
@@ -1907,6 +1909,9 @@ def clean(context: Context,
         raise tmt.utils.GeneralError(
             "Options --last and --id cannot be used together.")
 
+    if workdir_root and not Path(workdir_root).exists():
+        raise tmt.utils.GeneralError(f"Path '{workdir_root}' doesn't exist.")
```

Comment:
Just checking - is it expected to error out here? I'm thinking if user want to make sure some path that doesn't exist is "cleaned", a warning would suffice?

Reply:
yeah,it should be suffice in that case, but in that case,it won't hurt,well,so much,if we give an error.And the error would be for other cases, if we don't error out, tmt will continue the following steps,which will do nothing for users,but just waste their time,right?:)